lib/rubocop/cop/style/unneeded_percent_x.rb

```diff
@@ -8,7 +8,9 @@ class UnneededPercentX < Cop
         MSG = 'Do not use `%x` unless the command string contains backquotes.'
 
         def on_xstr(node)
-          add_offense(node, :expression) if node.loc.expression.source !~ /`/
+          return if node.loc.expression.source =~ /[`\n]/
```

Comment:
I'd rather count the lines of the source to avoid an embedded `\n` in the backquoted expression.

When doing code analysis one of the rules of thumb is to avoid using regexps unless totally necessary.